modules/dsfhub-azure-postgresql-flexible/main.tf

```diff
@@ -0,0 +1,36 @@
+terraform {
+  required_providers {
+    dsfhub = {
+      source = "imperva/dsfhub"
+    }
+  }
+}
+
+resource "dsfhub_data_source" "this" {
+  server_type = "AZURE POSTGRESQL FLEXIBLE"
+
+  admin_email               = var.admin_email
+  asset_display_name        = var.asset_display_name
+  asset_id                  = var.asset_id
+  audit_pull_enabled        = var.audit_pull_enabled
+  database_name             = var.database_name
+  gateway_id                = var.gateway_id
+  location                  = var.location
+  logs_destination_asset_id = var.logs_destination_asset_id
+  parent_asset_id           = var.parent_asset_id
+  server_host_name          = var.server_host_name
+  server_ip                 = null
+  server_port               = var.server_port
```

Comment:
Is it possible to connect on a different port or can we just stick 5432 in here?

Reply:
No, doesn't appear to let you choose. I'll remove it.